### PR TITLE
Use ngen64 on 64-bit

### PR DESCRIPTION
**Pull request by XhmikosR**
Notes:

1. Not sure what the current Inno Setup version is, but it looks like it's 6.x. The check should be adapted later

https://github.com/SubtitleEdit/subtitleedit/blob/8f3e5ae193b6f1fd8a74f665d7b43f1da8fdb020/installer/Subtitle_Edit_installer.iss#L25-L27

2. IIRC Inno Setup 6.x is only Unicode thus this check is probably moot:

https://github.com/SubtitleEdit/subtitleedit/blob/8f3e5ae193b6f1fd8a74f665d7b43f1da8fdb020/installer/Subtitle_Edit_installer.iss#L29-L31

**Untested!** Should work but someone should test it before landing it (and take care of the above TODO later)

### CODE REVIEW

File: installer/Subtitle_Edit_installer.iss [hunk 1]
```diff
@@ -897,15 +900,16 @@ function InitializeSetup(): Boolean;
...
-    if not WizardSilent() then 
+    if not WizardSilent() then
     begin
       if SuppressibleMsgBox(CustomMessage('msg_AskToDownNET'), mbCriticalError, MB_YESNO or MB_DEFBUTTON1, IDNO) = IDYES then
         ShellExec('open','https://go.microsoft.com/fwlink/?LinkId=2085155','','',SW_SHOWNORMAL,ewNoWait,ErrorCode);
         Result := False;
       end;
     end;
   end;
-end.
```
Comment (niksedk) [inline installer/Subtitle_Edit_installer.iss:911]: Hi @XhmikosR,

thx for the PR, works fine, except for the change in line 911 which give a compile error now.

Comment (XhmikosR) [reply installer/Subtitle_Edit_installer.iss]: @niksedk can you try now? I'm having trouble building the project since I'm using Inno Setup 6.x.

Comment (niksedk) [reply installer/Subtitle_Edit_installer.iss]: Works very well now :)

**Comment by niksedk** [comment]
thx :)

**Pull request merged**